Taegisxdrv2 investigation fields fixes (#29766)
demisto/content: Packs/SecureWorks/ReleaseNotes/5_0_3.md

#### Integrations

> ##### TaegisXDR v2

Reviewer: Please refer to https://xsoar.pan.dev/docs/documentation/release-notes#entity-styling:~:text=Entity%20Styling-,%23,-Command%20names%3A%20%2D%20should
For Xsoar best practice release notes.

> - Added support for **serviceDeskId**, **serviceDeskType**, and **tags** for investigation updates.

Reviewer: ```suggestion
- Added support for *serviceDeskId*, *serviceDeskType*, and *tags* for arguments in ***taegis-update-investigation*** command.
```

> - Added support for **assignee_user**, **alertsEvidence**, and **tags** when fetching incidents/investigations.

Reviewer: ```suggestion
- Added support for *assignee_user*, *alertsEvidence*, and *tags* when fetching incidents/investigations.
```

> - Added support for *assignee_user*, *alertsEvidence*, and *tags* when fetching incidents/investigations.

Reviewer: I didn't see any specifications for this in the yml file or the readme. 
Does this change only takes place in the query string?

Author: Yup, those are only for when querying data. The command `taegis-add-evidence-to-investigation` allows for adding alertsEvidence to an invesitgation.

> - Fixed an issue with the **page** and **perPage** query arguments submitted as a strings instead of integers.

Reviewer: ```suggestion
- Fixed an issue with the *page* and *perPage* query arguments submitted as strings instead of integers.
```
